src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoTest.java

```diff
@@ -300,23 +302,37 @@ private void ingestOmopTable(
 
   private Dataset stageOmopDataset() throws Exception {
     Dataset dataset = readDataset("omop/it-dataset-omop.json");
+    SnapshotBuilderSettings settings = readSettings("omop/settings.json");
     connectedOperations.addDataset(dataset.getId());
     bigQueryDatasetPdao.createDataset(dataset);
+    settingsDao.upsertSnapshotBuilderSettingsByDataset(dataset.getId(), settings);
 
     // Stage tabular data for ingest.
-    ingestOmopTable(dataset, "concept", "omop/concept-table-data.json", 3);
-    ingestOmopTable(dataset, "concept_ancestor", "omop/concept-ancestor-table-data.json", 2);
+    ingestOmopTable(dataset, "concept", "omop/concept-table-data.json", 4);
```

Comment:
The method is called `ingestOmopTable()` but it doesn't seem to be omop specific.

Reply:
True I could rename it to `ingestTable()`